less/dropdowns.less

```diff
@@ -212,6 +212,20 @@
   right: auto;
 }
 
+@media (max-width: @screen-sm-min) {
+  .dropdown-menu-bottom {
+    position: fixed;
+    bottom: 0;
+    top: auto;
+    width: 100%;
+    max-width: 100% !important;
+    max-height: 50% !important;
+    min-height: 200px;
+    box-shadow: 0 20px 66px 0 rgba(34, 48, 73, 1);
```

Comment:
Where does this colour come from?

Reply:
the colour is the current colour of the box-shadow but with 100% opacity so it adds a bit of a border on the top. @mobashagha suggested that